WSRequestHandler.cpp

```diff
@@ -1093,4 +1095,286 @@ void WSRequestHandler::HandleGetRecordingFolder(WSRequestHandler* req)
 
 	req->SendOKResponse(response);
 	obs_data_release(response);
-}
\ No newline at end of file
+}
+
+void WSRequestHandler::HandleGetTextGDIPlusProperties(WSRequestHandler* req)
+{
+
+	const char* itemName = obs_data_get_string(req->data, "source");
+	if (!itemName)
+	{
+		req->SendErrorResponse("invalid request parameters");
+		return;
+	}
+
+	const char* sceneName = obs_data_get_string(req->data, "scene-name");
+	obs_source_t* scene = Utils::GetSceneFromNameOrCurrent(sceneName);
+	if (scene == NULL) {
```

Comment:
Yes, my fault, I had bad C++ habits when I started obs-websocket...
As for that text copy thing, it's part of GitHub's code review and code commenting system. You can do something similar by leaving a GitHub comment on code (not a code comment) by clicking on the blue `+` button on the left of the currently hovered line.